lib/restforce/db/accumulator.rb

```diff
@@ -31,6 +31,19 @@ def attributes
         end
       end
 
+      # Public: Get a Hash representing the current values for the items in the
+      # passed Hash, as a subset of this Accumulator's attributes Hash.
+      #
+      # comparison - A Hash mapping of attributes to values.
+      #
+      # Returns a Hash.
+      def current(comparison)
+        attributes.each_with_object({}) do |(attribute, value), diff|
```

Comment:
Perhaps `diff` here should be named `final`?

Also: I think the `diff` method below is now not used?